database/migrations/2021_12_29_175805_add_release_remarks_to_departure_release_requests_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddReleaseRemarksToDepartureReleaseRequestsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('departure_release_requests', function (Blueprint $table) {
+            $table->string('remarks')
```

Comment:
Worth constraining the column length to the max length defined in the validator? Can't recall the default signature to `$table->string`. Might save us some space :D

Reply:
Laravel's default is 255 :) Which is incidentally what the plugin is constrained to :)